tests/testCommon.ml

```diff
@@ -1,101 +1,82 @@
-let alice_addr = Ligo.address_from_literal "alice_addr"
-let bob_addr = Ligo.address_from_literal "bob_addr"
-let leena_addr = Ligo.address_from_literal "leena_addr"
-let charles_key_hash = Ligo.key_hash_from_literal "charles_key_hash"
+open OUnit2
```

Comment:
Bah, it looks like the diff is rendering this a bit weirdly. This file is a new module, and `testLib.ml` is just the original `testCommon.ml` renamed.